# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fastfetch/hotfix-t2.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fastfetch-2.52.0/src/logo/ascii/t2.txt.vanilla	2025-09-05 17:41:43.098489851 +0200
+++ fastfetch-2.52.0/src/logo/ascii/t2.txt	2025-09-05 17:42:01.072821610 +0200
@@ -1,5 +1,5 @@
                         $3//$2
-                       $3//$2
+ $1TTTTTTTTTTTTTTT$2       $3//$2
  $1TTTTTTTTTTTTTTT$2      $3//$2 $1..::$2  $4:-:.$2
        $4ttt$2           $3//$2 $1.::.$2    $4.:-:$2
        $4ttt$2          $3//$2  $1.:.$2       $4--$2
